devops/DeployCloudformationS3Bucket/deployStackS3Bucket.py

```diff
@@ -1,32 +1,54 @@
-#Deploys a Stack and S3 Bucket To Cloudformation.
+#Deploys a stack and S3bucket to cloudformation.
+
+#Imports Python libraries
 import boto3
 import re
 import os
```

Comment:
os library not used, recommend removing

Reply:
@eknigge I removed the unused import os library. Thank you.